src/sig/sig.h

```diff
@@ -63,108 +84,123 @@ struct OQS_SIG {
 	size_t length_public_key;
 	/** The (maximum) length, in bytes, of secret keys for this signature scheme. */
 	size_t length_secret_key;
-	/** The (maximum) length, in bytes, of a signature for this signature scheme. */
-	size_t max_length_signature;
+	/** The (maximum) length, in bytes, of signatures for this signature scheme. */
```

Comment:
Let's take the (maximum) out of parentheses.

Reply:
I copied that from kem.h; we should keep or change both...